### ISSUE TITLE
Fix the favicon

### ISSUE THREAD

**Issue by nown1ne** [NITRR-Open-Source-Community/NOSC-Website#3]
**Description**
The favicon (image shown in the tab) is not clearly visible

**Difficulty**
Medium

**To Reproduce**
1. Go to 'nosc.netlify.app'
2. Notice the opened tab and observe the logo of NOSC

**Expected behavior**
The NOSC logo in the tab should be clearly visible

**Screenshots**
![image](https://github.com/NITRR-Open-Source-Community/NOSC-Website/assets/25835195/a03a2a24-3d96-4eca-9c2c-aa73d3ca8e08)

**Desktop (please complete the following information):**
 - OS: MacOs
 - Browser Edge
 - Version 117

**Additional context**
Nil

**Comment by sarafurd** [comment]
please assign this to me

**Comment by IshaanAdarsh** [comment]
Hi @sarafurd, I have already assigned you #6. We want a lot of people to participate in hacktoberfest using this project. I can't assign each issue to a singular person.

**Comment by suresh8919** [comment]
Hello @IshaanAdarsh  , I'm eager to contribute. Please assign me

**Comment by suresh8919** [comment]
I do have a couple of questions that I was hoping you could help clarify:

If I successfully complete an assigned task, would it be acceptable for me to submit a pull request on October 1st to ensure it counts toward my participation in Hacktoberfest? Or would this result in the task being unassigned?
Is there a specific timeframe within which assigned tasks must be completed to avoid potential reassignment to other contributors?

**Comment by IshaanAdarsh** [comment]
> I do have a couple of questions that I was hoping you could help clarify:
> 
> If I successfully complete an assigned task, would it be acceptable for me to submit a pull request on October 1st to ensure it counts toward my participation in Hacktoberfest? Or would this result in the task being unassigned? Is there a specific timeframe within which assigned tasks must be completed to avoid potential reassignment to other contributors?

yes sure, submit it on Oct 1. We are not too strict about the timeline.

**Comment by suresh8919** [comment]
Hello @IshaanAdarsh I have fixed the issue

**Comment by suresh8919** [comment]
If you want any extra thing to contribute please let me know

**Comment by IshaanAdarsh** [comment]
you can check out https://github.com/IshaanAdarsh/ezmail/issues, we are maintainers for that too

**Comment by suresh8919** [comment]
Hello @IshaanAdarsh Is my PR merged or not It is not showing for me Can you please respond I have fixed the favicon

**Comment by nown1ne** [comment]
![image](https://github.com/NITRR-Open-Source-Community/NOSC-Website/assets/25835195/8c230726-0db8-46c5-98c4-dfcc1e46390f)
Yes it has been merged

**Comment by One2sahu** [comment]
Sir I want to work on this issue please assign me this issue.
